### PR TITLE
Create thumbnail.gs

### PR DESCRIPTION
**Pull request by grant**
An example of creating a Google Slides thumbnail image.

### CODE REVIEW

File: slides/thumbnail/thumbnail.gs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Saves a thumbnail image of the current Google Slide presentation in Google Drive.
+ * Logs the image URL.
+ * @param {number} i The zero-based slide index. 0 is the first slide.
+ * @example saveThumbnailImage(0)
+ */
+function saveThumbnailImage(i) {
+  var presentation =  SlidesApp.getActivePresentation();
+  var thumbnail = Slides.Presentations.Pages.getThumbnail(presentation.getId(),
+                                                          presentation.getSlides()[i].getObjectId());
```
Comment (erickoledadevrel) [inline slides/thumbnail/thumbnail.gs]: Line length.

**Comment by erickoledadevrel** [comment]
Should this be added to `advanced/slides.gs` instead? Requires the advanced service to be enabled, which isn't clear in it's current location. Also don't think it makes sense to have a `/thumbnail/` directory, as it's hard to imagine there being multiple other thumbnail-related samples.

File: slides/thumbnail/thumbnail.gs (hunk 1 above)
Comment (grant) [reply slides/thumbnail/thumbnail.gs]: Reformatted.

**Comment by grant** [comment]
@erickoledadevrel PTAL.

**Pull request merged**